src/Lachain.Storage/DbCompact/DbShrinkRepository.cs

```diff
@@ -130,6 +133,45 @@ public void DeleteVersion(uint repository, ulong block, ulong version)
             );
         }
 
+        private void SetTimePassed(ulong timePassed)
```

Comment:
Should we save benchmark data in database?

Reply:
not sure about that. the process takes a lot of time. node can shut down in the middle, so I put it is db.

Reply:
ok,  let's keep it in database